adapters/repos/db/inverted/objects_test.go

```diff
@@ -515,6 +515,59 @@ func TestAnalyzeObject(t *testing.T) {
 			assert.ElementsMatch(t, expectedNumbers, actualNumbers, res)
 			assert.ElementsMatch(t, expectedUUID, actualUUID, res)
 		})
+
+		t.Run("when rep prop is stored as empty string slice", func(t *testing.T) {
+			uuid := "cf768bb0-03d8-4464-8f54-f787cf174c01"
+			name := "Transformers"
+			schema := map[string]interface{}{
+				"name":      name,
+				"reference": []string{},
+			}
+
+			props := []*models.Property{
+				{
+					Name:         "name",
+					DataType:     []string{"string"},
+					Tokenization: "word",
+				},
+				{
+					Name:     "reference",
+					DataType: []string{"SomeClass"},
+				},
+			}
+			res, err := a.Object(schema, props, strfmt.UUID(uuid))
+			require.Nil(t, err)
+
+			expectedUUID := []Countable{
+				{
+					Data:          []byte(uuid),
+					TermFrequency: 0,
+				},
+			}
+
+			expectedName := []Countable{
+				{
+					Data:          []byte(name),
+					TermFrequency: 1,
+				},
+			}
+
+			require.Len(t, res, 2)
+			var actualUUID []Countable
+			var actualName []Countable
+
+			for _, elem := range res {
+				switch elem.Name {
+				case "_id":
+					actualUUID = elem.Items
+				case "name":
+					actualName = elem.Items
+				}
+			}
+
+			assert.ElementsMatch(t, expectedUUID, actualUUID, res)
+			assert.ElementsMatch(t, expectedName, actualName, res)
```

Comment:
Why not do the assert directly in the for loop (or maybe even on res?)?
```

			for _, elem := range res {
				switch elem.Name {
				case "_id":
                                        assert.ElementsMatch(t, expectedUUID, elem.Items, res)
				case "name":
                                        assert.ElementsMatch(t, expectedName, elem.Items, res)
				}
			}
```

Reply:
Yeah, those suggestions _are_ other ways of accomplishing the same goal. However, this matches the style of all the other existing tests. Is there a benefit to your suggestions aside from personal preference?

Reply:
I just think that the current code is quite long for what it does and afaik lines 556-569 are equivalent to these two lines which are much clearer. For the current code I had to think for a few second what is happening there. 
```
assert.ElementsMatch(t, expectedUUID, res["_id"])
assert.ElementsMatch(t, expectedName, res["name"])
```

It's not important, so leave it like it is if you prefer that.